src/freertos_drivers/ti/TivaEEPROMFile.hxx

```diff
@@ -75,10 +75,44 @@ public:
     /// @return number of bytes written upon success, -errno upon failure
     ssize_t write(unsigned int index, const void *buf, size_t len) override
     {
-        HASSERT(index % 4 == 0);
-        HASSERT(len % 4 == 0);
-        MAP_EEPROMProgram((uint32_t *)buf, index + byteOffset_, len);
-        return len;
+        size_t count = 0;
+        uint8_t *b = (uint8_t *)buf;
+        // Partial write at the beginning.
+        if (index & 3)
```

Comment:
I prefer masks to be hex. Can we make this hex? More below.

Reply:
switched all masks to hex. There were a lot.